src/custom/utils/operator/error.ts

```diff
@@ -8,29 +13,23 @@ export enum ApiErrorCodes {
   InsufficientFunds = 'InsufficientFunds',
   InsufficientFee = 'InsufficientFee',
   UnsupportedToken = 'UnsupportedToken',
-  WrongOwner = 'WrongOwner'
+  WrongOwner = 'WrongOwner',
+  NotFound = 'NotFound',
```

Comment:
Is this something the backend returns?
This is just an enum with backend codes https://github.com/gnosis/gp-v2-services/blob/0bd5f7743bebaa5acd3be13e35ede2326a096f14/orderbook/openapi.yml#L562

Reply:
This is probably the new state added the other day https://github.com/gnosis/gp-v2-services/blob/b141b2bf32a0c861ef952d58290161abc3265b24/orderbook/src/fee.rs#L88

Guess they forgot to update open api docs